packages/jest-each/src/bind.js

```diff
@@ -107,8 +115,13 @@ const arrayFormat = (title, rowIndex, ...args) => {
   );
 };
 
-const applyRestParams = (params: Array<any>, test: Function) => {
-  if (params.length < test.length) return done => test(...params, done);
+const applyRestParams = (
+  supportsDone: boolean,
+  params: Array<any>,
+  test: Function,
+) => {
+  if (supportsDone && params.length < test.length)
+    return done => test(...params, done);
 
   return () => test(...params);
```

Comment:
nit: can we fold it to:
```js
return supportsDone && params.length < test.length 
  ? done => test(...params, done)
  : () => test(...params)
```
same suggestion for `applyObjectParams` below

Reply:
Like it! may as well drop the `return` too 😜 what do you think of this?

```js
const applyRestParams = (
  supportsDone: boolean,
  params: Array<any>,
  test: Function,
) => supportsDone && params.length < test.length
  ? done => test(...params, done)
  : () => test(...params);
```

Reply:
Yea, I usually have a lint rule to fix it for me :p